test/e2e/traffic_test.go

```diff
@@ -69,12 +69,15 @@ func TestTraffic(t *testing.T) {
 		})
 
 		ctx.NewSubTest("requests to c should fail").Run(func(ctx framework.TestContext) {
-			a[0].CallOrFail(t, echo.CallOptions{
+			res, err := a[0].Call(echo.CallOptions{
```

Comment:
Looking from n00b perspective - can we give `a` a meaningful name, or maybe even better, if possible, extract `a[0]` to  its own variable (thinking of `serviceA`)?

Reply:
If it's a known convention there's no reason to introduce something else other than making it confusing.